docs/runtimes/application.ipynb

```diff
@@ -82,16 +82,301 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        image="your-image",  # should include mlrun for importing V2ModelServer
</pre><p>image can be removed. default is mlrun/mlrun and its not really relevant for the example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>